docs/guidance/map/awards_contracts.md

```diff
@@ -145,36 +145,48 @@ There are no identifiers for the individual supplier/value pairings on the origi
 
 [View the example in Paraguay's API](https://contrataciones.gov.py/datos/api/v3/doc/ocds/record/ocds-03ad3f-340885-1)
 ```
+
 ## Purchase orders
 
 A purchase order is a specific type of contract, an official document issued by a buyer committing to pay a supplier for the supply of specific goods, services or works to be delivered in the future.
 
 Purchase orders can be issued against an existing contract, or if no prior contract exists then acceptance of a purchase order by a supplier forms a contract between buyer and supplier.
 
-Purchase orders that are made against contracts with a definite quantity or value of items ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting items on the purchase order and the contract it relates to.
+Purchase orders that are made against contracts with a definite quantity or value of items ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting items on the purchase order and the contract it relates to. Instead they ought to be disclosed in the `implementation.transactions` array of the `Contract`.
 
-### Example: Double counting contracts and purchase orders
+### Example: Purchase orders
 
-The UK's Department for Transport awards a £1.2m, 12-month contract to KPMG to provide the Project Management Office function for a project to construct a new highway bypass. The contract specifies that payment will be made quarterly in arrears in four equal amounts. The contract is represented in the `contracts` section of OCDS as follows:
+The Bureau of Parks and Recreation in Portland, Oregon, USA, awards a 2 year contract worth up to $1m to a local contractor to maintain Mt. Tabor Park. This contract specifies the goods, services and works to be provided along with unit prices and estimates of quantities. The contract is represented in the `contracts` section of OCDS as follows:
 
 ```{csv-table-no-translate}
 :header-rows: 1
 :file: ../../examples/purchase_orders/parent_contract.csv
 ```
 
-Calculating the sum of the contract value in the above example gives the correct result of £1.2m.
+The contract in this scenario is treated as a type of [framework agreement](framework_agreements.md), with the contract value being disclosed in the `maximumValue` field instead of the `value` field.
```

Comment:
If this indefinite-quantity/indefinite-delivery contract is treated as a framework agreement per https://github.com/open-contracting/standard/issues/1694#issuecomment-2483336606, it should appear only in the awards array, according to the [guidance on first-stage selection](https://standard.open-contracting.org/staging/1.2-dev/en/guidance/map/framework_agreements/#first-stage-selection-and-addition-of-suppliers-to-the-framework-agreement).

Also, each purchase order would be treated as the [award of a procurement contract without second-stage competition](https://standard.open-contracting.org/staging/1.2-dev/en/guidance/map/framework_agreements/#award-of-a-procurement-contract-without-second-stage-competition) and would be modelled as an `Award` in a separate contracting process.

Is there an example of purchase orders outside of a framework agreement?

Edit: I haven't reviewed beyond this point.